src/nlua_debug.c

```diff
@@ -0,0 +1,74 @@
+/*
+ * See Licensing and Copyright notice in naev.h
+ */
+
+/**
+ * @file nlua_debug.c
+ *
+ * @brief Handles the Lua debug bindings.
+ *
+ * These bindings control the debug flags.
+ */
+
+/** @cond */
+#include "naev.h"
+/** @endcond */
+
+#include "nlua_debug.h"
+
+#include "debug.h"
+#include "nluadef.h"
+
+/* Debug metatable methods. */
+
+static int debugL_dispEmitters( lua_State *L );
+static const luaL_Reg debugL_methods[] = {
+   { "dispEmitters", debugL_dispEmitters },
+   {0,0}
+}; /**< Debug metatable methods. */
+
+
+/**
+ * @brief Loads the Debug library.
+ *
+ *    @param env Environment to load Debug library into.
+ *    @return 0 on success.
+ */
+int nlua_loadDebug( nlua_env env )
+{
+   /* Register the values */
+   nlua_register(env, "debug", debugL_methods, 0);
+
+   return 0;
+}
+
+
+/**
+ * @brief Toggles the emitter marker.
+ *
+ * @usage dispEmitters() -- Trail emitters are marked with crosses.
+ * @usage dispEmitters(false) -- Remove the markers.
+ *
+ *    @luatparam[opt=true] boolean state Whether to set or unset markers.
+ * @luafunc dispEmitters
+ */
+static int debugL_dispEmitters( lua_State *L )
```

Comment:
why not just have the default be false and do
```
if (lua_toboolean(L,1))
   debug_setFlag(MARK_EMITTER);
else
   debug_rmFlag(MARK_EMITTER);
```?

Reply:
It doesn't feel right to me to be false by default. Most people will use it to activate.

Reply:
It's fine in this case. However, in general, I'm trying to have boolean parameters default to false, as it avoids the `lua_gettop(L)`-dependent behavior. I don't like having `my_func()` and `my_func(nil)` behave differently, as I think it can be a source of bugs in some cases.